ttsai/rotator_verification.ipynb

```diff
@@ -0,0 +1,3831 @@
+{
```

Comment:
<p>Please add to all step messages like:</p><p>In the preamble</p><pre class="ql-syntax" spellcheck="false">start_time = datetime.now()
test_message = "Rotator New Trjectory Verification Test"
script = salobj.Controller("Script", index=10)
await asyncio.sleep(10) #wait 10 second may help with DDS problems; closing all other kernels may help too
print(f'time to start is {datetime.now() - start_time} [s]')
</pre><p>and before each test step:</p><pre class="ql-syntax" spellcheck="false">now = datetime.now()
script.log.info(f"START -- {test_message} -- LVV-T2539 -- Test step desciption -- Starting Time: {now} UTC")
</pre><p>And after each test step:</p><pre class="ql-syntax" spellcheck="false">now = datetime.now()
script.log.info(f"START -- {test_message} -- LVV-T2539 -- Test step desciption -- Stopping Time: {now} UTC")
</pre><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/lsst-ts/ts_notebooks/pull/72/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
Updated in the commit: 137c4fd to f4b262b already. There is the case-sensitive error in your folder and I could not reach the LSP at summit and Tucson Teststand to rewrite the history. Stay like this at this time. Thanks!